MantidQt/CustomInterfaces/src/Tomography/TomographyROIViewQtWidget.cpp

```diff
@@ -1237,22 +1296,72 @@ void ImageROIViewQtWidget::updateColorMap(const std::string &filename) {
   //  updateColorMap(m_colorMapFilename);
   m_ui.colorBarWidget->setViewRange(1, 65536);
 
-  m_presenter->notify(IImageROIPresenter::ColorRangeUpdated);
+  m_presenter->notify(ITomographyROIPresenter::ColorRangeUpdated);
 }
 
 /**
  * Slot for the signal emitted by the color bar widget when there's an
  * update in the values
  */
-void ImageROIViewQtWidget::colorRangeChanged() {
+void TomographyROIViewQtWidget::colorRangeChanged() {
   // the presenter should handle the image display update
-  m_presenter->notify(IImageROIPresenter::ColorRangeUpdated);
+  m_presenter->notify(ITomographyROIPresenter::ColorRangeUpdated);
 }
 
-void ImageROIViewQtWidget::closeEvent(QCloseEvent *event) {
-  m_presenter->notify(IImageROIPresenter::ShutDown);
+void TomographyROIViewQtWidget::closeEvent(QCloseEvent *event) {
+  m_presenter->notify(ITomographyROIPresenter::ShutDown);
   event->accept();
 }
 
+void TomographyROIViewQtWidget::findCORClicked() {
+  // this should run a --find-cor, empty executable path string signifies that
+  // the default external interpretor will be used
+  emit(findCORClicked("", {"--find-cor"}));
+}
+
+void TomographyROIViewQtWidget::readCoRFromProcessOutput(const QString &str) {
+  if (str.isEmpty()) {
+    // the process string is empty, it is likely the process crashed or was
+    // unable to execute properly
+    return;
```

Comment:
Should we maybe print a warning to the log as well as I'm guessing this is pretty exceptional?